batches/aapb-collaboration-7-leftover.txt

```diff
@@ -1,4 +1,13 @@
-# see https://github.com/clamsproject/aapb-annotations/issues/79 for more info
+# --------------------------------------------------------------------------------
+# Originally, a set of about 4000 videos was selected for slate annotation. This 
+# batch has the GUIDs that were not include in batch aapb-collaboration-7, which
+# contains the GUIDs of assets that were annotated.
+# 
+# See https://github.com/clamsproject/aapb-collaboration/issues/79 for more info.
```

Comment:
looks like there's no issue 79, and issue 7 in the collab repo seems to be the right one.

Reply:
Yup, seems to be a typo on my end.